__tests__/setupTests.js

```diff
@@ -1,3 +1,7 @@
 // react-testing-library renders your components to document.body,
 // this adds jest-dom's custom assertions
 import "@testing-library/jest-dom";
+import { TextDecoder, TextEncoder } from "util";
```

Comment:
Added because JOSE package is written in commonjs and it's submodules need to be imported for jest